torch/utils/data/dataloader.py

```diff
@@ -23,6 +23,14 @@
 
 import torch.utils.data.graph_settings
 
+__all__ = [
+    "DataLoader",
+    "_DatasetKind",
```

Comment:
@albanD says this one will raise an error in CI, because this has a `_` in front.

We should remove it. Removing it won't prevent us from using it from elsewhere.

Reply:
Addressed